devdoc/umockcallrecorder_requirements.md

```diff
@@ -200,3 +201,24 @@ extern int umockcallrecorder_fail_call(UMOCKCALLRECORDER_HANDLE umock_call_recor
 **SRS_UMOCKCALLRECORDER_01_050: [** If index is invalid, umockcallrecorder_fail_call shall return a non-zero value. **]**
 
 **SRS_UMOCKCALLRECORDER_01_051: [** If umockcall_set_fail_call fails, umockcallrecorder_fail_call shall return a non-zero value. **]**
+
+
+## umockcallrecorder_can_call_fail
+
+```c
+extern int umockcallrecorder_can_call_fail(UMOCKCALLRECORDER_HANDLE umock_call_recorder, size_t index, int* can_call_fail);
+```
+
+
+**SRS_UMOCKCALLRECORDER_31_056: [** If umock_call_recorder or call_can_fail is NULL, umockcallrecorder_can_call_fail shall return a non-zero value. **]**
+
+**SRS_UMOCKCALLRECORDER_31_057: [** If index is invalid, umockcallrecorder_can_call_fail shall return a non-zero value. **]**
+
+**SRS_UMOCKCALLRECORDER_31_057: [** umockcallrecorder_can_call_fail shall determine whether given call can fail or not by calling umockcall_set_fail_call. **]**
```

Comment:
>se [](start = 140, length = 2)

get

Reply:
Also I used comment #57 twice.  Fixed too.

---
In reply to: [224577211](https://github.com/Azure/umock-c/pull/44#discussion_r224577211) [](ancestors = 224577211)